tests/drivers/spi/spi_loopback/boards/nucleo_h743zi.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_SPI_LOOPBACK_DRV_NAME="SPI_1"
+
+CONFIG_SPI_STM32_DMA=n
```

Comment:
This should not be required as DMA is default n.

Reply:
correct